### ISSUE TITLE
Add a parameter to control the showing of the trailing zero

### ISSUE THREAD

**Issue by sleepingkit** [entronad/number_display#3]
Hope that there is a boolean that can control the showing of the trailing zero.
For example 100.12
Now is when I set 3 decimal places, it will show 100.12,
If I set show trailing zero, I hope it will show 100.120

**Comment by cyrilhl** [comment]
Agree, sometime client want to align the format to 2 decimal no matter the value have decimal value or not.
